### PR TITLE
Safari TP CSS Properties and Values partial support

### PR DESCRIPTION
**Pull request by chicoxyzzy**
Safari TP 67 has added CSS Properties and Values partial support under Experimental features flag
https://webkit.org/blog/8419/release-notes-for-safari-technology-preview-67/

image

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -128,8 +128,9 @@
         "has_details": true
       },
       "safari": {
-        "completeness": "development",
-        "has_details": true
+        "completeness": "partially",
+        "has_details": true,
+        "since":  "Safari TP"
```
Comment (surma) [inline data.json]: Can you write the version here, TP is an ever-changing target ^^

**Review by surma — CHANGES_REQUESTED** [review]
Thanks very much for the timely PR! Just one nit, then this is good to go

File: data.json (hunk 1 above)
Comment (chicoxyzzy) [reply data.json]: ✅

**Comment by surma** [comment]
🎉

**Review by surma — APPROVED** [review]

**Pull request merged**